bin/moodle-start

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+bin/moodle-docker-compose up -d
+bin/moodle-docker-wait-for-db
```

Comment:
1. I'm not very convinced of the value of this `moodle-start` script. My '_philosophy_' when originally designing this project was just to add the 'minimum helpers' to get by for convenience and scriptability, but encourage people to use the docker compose commands when necessary. This seems to abstracting what's going on under the hood more than I think this project should. For most people, doing `moodle-docker-compose up -d` will be enough for their workflow of development.

2. I don't think this script is well named.

Reply:
I understand your philosophy and removed the script.